src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_InbreedingCoeff.java

```diff
@@ -121,47 +55,39 @@ public void initialize ( AnnotatorCompatible walker, GenomeAnalysisEngine toolki
     public List<VCFInfoHeaderLine> getDescriptions() { return Collections.singletonList(GATKVCFHeaderLines.getInfoLine(getKeyNames().get(0))); }
 
     @Override
-    public Map<String, Object> annotate(final RefMetaDataTracker tracker,
-                                        final AnnotatorCompatible walker,
-                                        final ReferenceContext ref,
-                                        final Map<String, AlignmentContext> stratifiedContexts,
+    public Map<String, Object> annotate(final ReferenceContext ref,
                                         final VariantContext vc,
                                         final Map<String, PerReadAlleleLikelihoodMap> perReadAlleleLikelihoodMap ) {
+        Utils.nonNull(vc);
+        final HeterozygosityUtils heterozygosityUtils = new HeterozygosityUtils(vc);
```

Comment:
Rename to something that doesn't end in `Utils`, since it looks bizarre to be instantiating a "utils" class like this.